python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -16,7 +17,7 @@
 # entry/init points.
 logger = logging.getLogger(__name__)
 
-DEFAULT_MAX_BUFFER_SIZE = int(100 * 1e6)  # 100 mB
```

Comment:
this was a bug

Reply:
I think there's a fundamental fix. I will fix it in a separate PR

Reply:
Did you mean the buffer size should be 1MB? If that's the case, can you update comment as well?